Binary files ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/apps/msplazer/.msplazer.cpp.swp and ./sandbox/my_sandbox/apps/msplazer/.msplazer.cpp.swp differ
Binary files ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/apps/msplazer/.msplazer.h.swp and ./sandbox/my_sandbox/apps/msplazer/.msplazer.h.swp differ
diff -u -r -N -x '*.o' -x Thumbs.db -x .DS_Store -x CMakeCache.txt -x misc/seqan_instrumentation/userdata/id.txt -x /home/bluebelly/Development/seqan-trunk/misc/seqan_instrumentation/userdata/id.txt -x misc/seqan_instrumentation/userdata/7uqc7v3iz0sdm7d7_stats.txt -x /home/bluebelly/Development/seqan-trunk/misc/seqan_instrumentation/userdata/7uqc7v3iz0sdm7d7_stats.txt -x .svn -x build -x util -x misc -x docs -x misc/seqan_instrumentation/bin -x /home/bluebelly/Development/seqan-trunk/misc/seqan_instrumentation/bin -x misc/seqan_instrumentation/last_revision_copy -x /home/bluebelly/Development/seqan-trunk/misc/seqan_instrumentation/last_revision_copy -x misc/seqan_instrumentation/userdata -x /home/bluebelly/Development/seqan-trunk/misc/seqan_instrumentation/userdata ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/demos/samio_demo.cpp ./sandbox/my_sandbox/demos/samio_demo.cpp
--- ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/demos/samio_demo.cpp	2011-09-13 14:52:47.000000000 +0200
+++ ./sandbox/my_sandbox/demos/samio_demo.cpp	2011-09-13 15:04:32.281756551 +0200
@@ -60,7 +60,7 @@
 		BamAlignmentRecord record;
 		record.qName = "QeryName";
 		record.flag = 0;
-		record.rId = 1;
+		record.rId = 0;
 		record.pos = 10;
 		record.mapQ = 255;
 		record.bin = 0;
Binary files ./misc/seqan_instrumentation/last_revision_copy/sandbox/my_sandbox/demos/.samio_demo.cpp.swp and ./sandbox/my_sandbox/demos/.samio_demo.cpp.swp differ
